pom.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.redhat.labs</groupId>
+	<artifactId>lodestar-artifacts</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
+	<properties>
+		<compiler-plugin.version>3.8.1</compiler-plugin.version>
+		<maven.compiler.parameters>true</maven.compiler.parameters>
+		<maven.compiler.source>11</maven.compiler.source>
+		<maven.compiler.target>11</maven.compiler.target>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<quarkus-plugin.version>1.13.7.Final</quarkus-plugin.version>
+		<quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
+		<quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+		<quarkus.platform.version>1.13.7.Final</quarkus.platform.version>
+		<surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
+		<wiremock.version>2.28.1</wiremock.version>
+		<javers.core.version>6.2.2</javers.core.version>
+		<sonar.host.url>https://sonarcloud.io</sonar.host.url>
+		<sonar.project.key>rht-labs_lodestar-activity</sonar.project.key>
```

Comment:
key should be - rht-labs_lodestar-artifacts

Reply:
updated